samples/highcharts/website/static-chart/demo.details

```diff
@@ -0,0 +1,7 @@
+---
+ name: Highcharts Static Chart for Highcharts Product Page
+ authors:
+   - Nancy Dillon
+ js_wrap: b
+ skipTest: true
```

Comment:
Unless there's a good reason, remove `skipTest: true` so that the static charts become part of the visual test routine.